<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-OUMFU5K" failures="1" tests="1" name="edge" time="25.547" errors="0" timestamp="2023-05-30T11:05:20 IST">
  <testcase classname="com.banking.steller.Openaccounttest" name="openacc" time="20.462">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at com.banking.steller.Openaccounttest.openacc(Openaccounttest.java:148)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- openacc -->
</testsuite> <!-- edge -->
